xbmc/GUIInfoManager.cpp

```diff
@@ -7174,7 +7174,7 @@ bool CGUIInfoManager::GetBool(int condition1, int contextWindow, const CGUIListI
       bReturn = g_application.m_pPlayer->GetPlaySpeed() < 0;
       break;
     case PLAYER_FORWARDING:
-      bReturn = g_application.m_pPlayer->GetPlaySpeed() > 1.5;
+      bReturn = g_application.m_pPlayer->GetPlaySpeed() > IPlayer::MaxPlaySpeed();
```

Comment:
actually I've update `IPlayer::MaxPlaySpeed()` to return `1.5`, so should I add `+ 0.05` like I've done at the other places?